### ISSUE TITLE
Flag -d

### ISSUE THREAD

**Issue by Sergey-Galan** [sveinbjornt/hear#28]
Hi, thanks for the work. I understand correctly? Will it work with the flag -d only the language package that is in ‘Siri’ and is loaded? I got a job only with the flag -d ‘ru-RU’ and ‘en -US’, but I need ‘uk-UA’. ‘Siri’ does not speak Ukrainian and so I can’t get a translation with a flag -d. I conducted an experiment, installed ‘Siri’ French, the language package loaded and I had an ‘fr -Fr’ Local with a flag -d.

**Comment by sveinbjornt** [comment]
I'm not sure I understand what you mean. You can view supported locales using the -l flag. In some cases, you will need to wait while your OS downloads the files required for the specified locale before you can start speech recognition for it.

**Comment by Sergey-Galan** [comment]
Yes, there is support for 'uk-UA', but only without the -d flag. And has a limit of approximately 100 words. With the -d flag (2023-12-17 15:39:33.697 hear[19913:128391] Required assets are not available for Locale:uk-UA
Error: On-device recognition is not supported for locale 'uk-UA’). After experimenting, I came to the conclusion that only those locales that Siri has work with the -d flag.

**Comment by sveinbjornt** [comment]
Noted in the documentation. This is an Apple limitation that has nothing to do with the command line tool as such.
